src/examples/example_client.cpp

```diff
@@ -32,12 +32,9 @@ int main(int argc, char** argv)
 {
   try
   {
-    //std::string endpoint = "opc.tcp://192.168.56.101:48030";
-    //std::string endpoint = "opc.tcp://user:password@192.168.56.101:48030";
-    std::string endpoint = "opc.tcp://127.0.0.1:4841/freeopcua/server/";
-    //std::string endpoint = "opc.tcp://localhost:53530/OPCUA/SimulationServer/";
```

Comment:
Do not remove these comments, they document how to connecto to different servers